specs/_features/eip7594/das-core.md

```diff
@@ -142,9 +142,8 @@ def recover_matrix(cells_dict: Dict[Tuple[BlobIndex, CellID], Cell], blob_count:
     for blob_index in range(blob_count):
         cell_ids = [cell_id for b_index, cell_id in cells_dict.keys() if b_index == blob_index]
         cells = [cells_dict[(blob_index, cell_id)] for cell_id in cell_ids]
-        cells_bytes = [[bls_field_to_bytes(element) for element in cell] for cell in cells]
 
-        all_cells_for_row = recover_all_cells(cell_ids, cells_bytes)
+        all_cells_for_row = [coset_evals_to_cell(cell) for cell in recover_all_cells(cell_ids, cells)]
```

Comment:
This doesn't seem right in terms of naming: `coset_evals_to_cell(cell) for cell in recover_all_cells(cell_ids, cells)`

You shouldn't feed a `cell` into `coset_evals_to_cell`.

Reply:
Kev fixed it here [169cc83](https://github.com/ethereum/consensus-specs/pull/3701/commits/169cc830d686a339208d92f3bdd4b18f990a5d1f)